harvard-university-of-bath.csl

```diff
@@ -449,6 +490,19 @@
           </group>
           <text prefix=" " suffix="." macro="publisher"/>
         </else-if>
+        <else-if type="entry">
+          <group prefix=" " suffix="." delimiter=". ">
+            <text macro="byeditor"/>
+            <text macro="edition"/>
+            <group delimiter=" ">
+              <text variable="title" font-style="italic"/>
+              <text macro="online"/>
+            </group>
+            <text macro="series"/>
+            <text macro="status"/>
+            <text macro="publisher"/>
+          </group>
+        </else-if>
```

Comment:
@alex-ball, I cleaned up the use of groups here. Output should be the same, but I edited this by hand without testing, so let me know if I made a mistake here.